oggm/utils.py

```diff
@@ -56,6 +56,8 @@
 logger = logging.getLogger(__name__)
 
 SAMPLE_DATA_GH_REPO = 'OGGM/oggm-sample-data'
+SAMPLE_DATA_COMMIT = '4ccda1dbcd37ecdabe1b80854b244905f51c6c65'
```

Comment:
I know how this goes but for future developers: could you add a short comment with `#` to explain how you obtain this string from the repository? Thanks!